Modules/Invoices/Controllers/InvoicesController.php

```
@@ -185,11 +173,11 @@ public function download(string $invoice): Response
      */
     public function view(int $invoiceId): View
     {
-        $invoice = Invoice::with(['client', 'user', 'invoiceGroup', 'items', 'taxRates', 'payments'])
-            ->findOrFail($invoiceId);
+        $invoice = $this->invoiceService->findWithRelationsOrFail($invoiceId, 
+            ['client', 'user', 'invoiceGroup', 'items', 'taxRates', 'payments']);
 
         // Get custom fields and values
-        $fields       = InvoiceCustom::where('invoice_id', $invoiceId)->get();
+        $fields       = InvoiceCustom::query()->where('invoice_id', $invoiceId)->get();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$fields'. (undefined)

(UnusedLocalVariable)